api/openapi-spec/swagger.json

```diff
@@ -4451,6 +4451,10 @@
    },
    "v1.Firmware": {
     "properties": {
+     "serial": {
+      "description": "Serial number",
```

Comment:
Does this need some validation (length, valid characters)?

One nit: could you make the description a little less minimal?

Reply:
Thanks for the quick review @rmohr !
Good point, I see that in oVirt we validated its length so I'll add such a validation.
As for the description, I used the one [libvirt provides](https://libvirt.org/formatdomain.html) :) But yeah, I'll change it.